group/assignment6/RearrangingCars.java

```diff
@@ -0,0 +1,68 @@
+package edu.codeU.assignment6;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RearrangingCars {
+
+    /**
+     * N - number of cars. N + 1 - number of parking places.
+     */
+    private int n;
+    /**
+     * Start configuration: at i_th position in configuration store i_th car parking position.
+     */
+    private int[] startCarsPosition;
+    /**
+     * Final configuration: at i_th position in configuration store i_th car parking position.
+     */
+    private int[] finalCarsPosition;
+    /**
+     * Array to store for each place car in start configuration.
+     */
+    private int[] startPlacePosition;
+    /**
+     * Array to store for each place car in final configuration.
+     */
+    private int[] finalPlacePosition;
+
+    public RearrangingCars(int n, int[] startPosition, int[] finalPosition) {
```

Comment:
I think you can get n from the size of startPosition, so you wouldn't need it as a paramter.